src/output/writeCGNSSurface.F90

```diff
@@ -1320,7 +1320,8 @@ subroutine writeSurfaceSol
                call storeSurfsolInBuffer(ind, buffer, jj, ii,       &
                     faceID, cellRange(1,1,kk), &
                     solNames(mm),              &
-                    viscousSubface, storeRindLayer)
+                    viscousSubface, storeRindLayer,&
+                    iBeg, iEnd+1, jBeg, jEnd+1)
```

Comment:
Is there a reasoning (other than convenience) for adding the 1s here to i/j end here instead of the low level checks in the subroutine itself? The additions here can cause confusion; the i/j beg/end variables have the same names here and the subroutine you are calling here, but the values will be different. I would rather put the +1s right when the variables are actually used, and maintain the same values for these variables which have the same names. Just a convenience thing.